### PR TITLE
Screen: Warned Persons (EXPOSUREAPP-4511)

### PR DESCRIPTION
**Pull request by jurajkusnier**
- Implementation of Key Submissions Card

### CODE REVIEW

File: Corona-Warn-App/src/main/res/layout/home_statistics_cards_keysubmissions_layout.xml [hunk 1]
```diff
@@ -1,24 +1,154 @@
...
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/card_padding"
+            android:layout_marginBottom="@dimen/spacing_tiny"
+            android:text="@string/statistics_card_submission_bottom_text"
+            android:textSize="@dimen/font_small"
+            app:layout_constraintBottom_toBottomOf="parent" />
+
+        <ImageButton
+            android:id="@+id/statistics_card_info_button"
+            style="@style/Widget.AppCompat.ImageButton"
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/main/res/layout/home_statistics_cards_keysubmissions_layout.xml:142]: There is a style for this

Comment (jurajkusnier) [reply Corona-Warn-App/src/main/res/layout/home_statistics_cards_keysubmissions_layout.xml:142]: true, I'll update styles

**Review by janetback — APPROVED** [review]
UA approved

**Comment by chiljamgossow** [comment]
device-2021-01-19-101515
 The primary value and label seem incorrect. The spacing on the bottom seems to be off.

File: Corona-Warn-App/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -402,12 +402,12 @@
         <item name="android:background">@drawable/contact_diary_card_ripple</item>
     </style>
 
-    <style name="StatisticsCardInfoButton" parent="buttonIcon">
-        <item name="android:height">@dimen/circle_icon_big</item>
-        <item name="android:width">@dimen/circle_icon_big</item>
+    <style name="StatisticsCardInfoButton" parent="@style/Widget.AppCompat.ImageButton">
+        <item name="android:layout_width">@dimen/radius_info_button</item>
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/main/res/values/styles.xml:406]: Please leave it as it is, so all image buttons have the same ripple

**Comment by jurajkusnier** [comment]
> device-2021-01-19-101515
> The primary value and label seem incorrect. The spacing on the bottom seems to be off.

**Pull request closed without merging**